solr/core/src/java/org/apache/solr/handler/component/HttpShardHandlerFactory.java

```diff
@@ -305,16 +306,16 @@ public void init(PluginInfo info) {
             sb);
     int soTimeout =
         getParameter(args, HttpClientUtil.PROP_SO_TIMEOUT, HttpClientUtil.DEFAULT_SO_TIMEOUT, sb);
-
-    this.defaultClient =
+    this.httpClientBuilder =
```

Comment:
please name `httpSolrClientBuilder` -- so as to clarify this isn't an Apache/Jetty HttpClient; it's a SolrClient.